### PR TITLE
[DOC-403] Added PG CVE table

### PR DESCRIPTION
**Pull request by aishwarya24**
https://deploy-preview-22913--infallible-bardeen-164bc9.netlify.app/preview/secure/vulnerability-disclosure-policy/#postgresql-security-vulnerabilities-in-postgresql-11-2

### CODE REVIEW

File: docs/content/preview/secure/vulnerability-disclosure-policy.md [hunk 1]
```diff
@@ -72,3 +72,43 @@ An important part of Yugabyte's strategy for building a secure platform for our
 Our [release notes](/preview/releases/ybdb-releases/) contain up-to-date information on security vulnerabilities and available patches.
 
 {{< /note >}}
+
+YugabyteDB relies on PostgreSQL, and the following CVEs have been addressed in the PostgreSQL code base.
+
+### PostgreSQL security vulnerabilities in PostgreSQL 11.2
```
Comment (ddhodge) [inline docs/content/preview/secure/vulnerability-disclosure-policy.md:78]:
```suggestion
### Vulnerabilities in PostgreSQL 11.2
```

File: docs/content/preview/secure/vulnerability-disclosure-policy.md [hunk 2]
```diff
@@ -72,3 +72,43 @@ An important part of Yugabyte's strategy for building a secure platform for our
 Our [release notes](/preview/releases/ybdb-releases/) contain up-to-date information on security vulnerabilities and available patches.
 
 {{< /note >}}
+
+YugabyteDB relies on PostgreSQL, and the following CVEs have been addressed in the PostgreSQL code base.
```
Comment (ddhodge) [inline docs/content/preview/secure/vulnerability-disclosure-policy.md:76]: put this sentence after the heading below

**Review by proff4eg — APPROVED** [review]
LGTM

File: docs/content/preview/secure/vulnerability-disclosure-policy.md [hunk 3]
```diff
@@ -57,18 +57,57 @@ An important part of Yugabyte's strategy for building a secure platform for our
...
+| YugabyteDB | {{<cve "CVE-2023-0574">}} | from v2.0 through [v2.13.0.0](/preview/releases/ybdb-releases/end-of-life/v2.13/) | {{<release "2.14.0.0">}} | Resolved |
+| YugabyteDB | {{<cve "CVE-2022-37397">}} | v2.6.1.0 | v2.6.1.1 | Resolved |
 
 {{< note title="Note" >}}
 
 Our [release notes](/preview/releases/ybdb-releases/) contain up-to-date information on security vulnerabilities and available patches.
 
 {{< /note >}}
+
+### Supply-chain vulnerabilities in YugabyteDB Postgres Query Layer (YSQL)
```
Comment (interstellarPotato) [inline docs/content/preview/secure/vulnerability-disclosure-policy.md:76]: I would add more details about the scope of this section. Example:

### Scope of Supply Chain Vulnerabilities

Please note that this policy covers only vulnerabilities in the query layer of PostgreSQL. We do not publish or disclose any other third-party (supply chain) vulnerabilities. Our focus is solely on ensuring the security and integrity of the PostgreSQL query layer.

**Review by interstellarPotato — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**